resolvers/webpack/index.js

```diff
@@ -265,7 +266,7 @@ function findExternal(source, externals, context) {
 
   // else, vanilla object
   for (var key in externals) {
-    if (!externals.hasOwnProperty(key)) continue
+    if (!has(externals, key)) continue
```

Comment:
Fair points. It's not a huge deal either way, or it wouldn't be so easy to have this back and forth. 😄

I hear what you're saying, and maybe one day all our key checks turn into `has` calls, but I don't think today is that day. 

I didn't realize the `has` part was a separate commit -- I'll just go ahead and merge the first one.